hw/mcu/stm/stm32f1xx/include/mcu/stm32_hal.h

```diff
@@ -62,6 +62,13 @@ struct stm32_hal_spi_cfg {
 #include "stm32f1xx_hal_uart.h"
 #include "mcu/stm32f1_bsp.h"
 
+/* hal_timer */
+#include "stm32f1xx_hal_tim.h"
+
+#define STM32_HAL_TIMER_MAX     (3)
+#define STM32_OFLOW_VALUE       (0x10000UL)
+#define STM32_NSEC_PER_SEC      (1000000000UL)
```

Comment:
I would keep STM32_OFLOW_VALUE and STM32_NSEC_PER_SEC inside c-file. They are of no use as an exported API.